middleware/authz.go

```diff
@@ -0,0 +1,41 @@
+package middleware
+
+// This plugin is based on Casbin: an authorization library that supports ACL, RBAC, ABAC
+// View source at:
+// https://github.com/hsluoyz/casbin
+
+import (
+	"net/http"
+
+	"github.com/hsluoyz/casbin"
```

Comment:
thanks @hsluoyz - I've added your pkg to the list in 898d761524954def9191dc8bea69191442581385 cheers!